_main.cfg

```diff
@@ -10,6 +10,9 @@
     type_arcane_focus= _ "type^arcane focus"
     type_electric= _ "type^electric"
     type_energy= _ "type^energy"
+    type_natural= _ "type^natural"
+    type_water= _ "type^water"
+    type_holy= _ "type^light"
```

Comment:
can this be `type_light=`?
I see it as light.png in the icon too.

Reply:
Fixed.